oracle/reactor.go

```diff
@@ -124,21 +131,60 @@ func (oracleR *Reactor) Receive(e p2p.Envelope) {
 	oracleR.Logger.Debug("Receive", "src", e.Src, "chId", e.ChannelID, "msg", e.Message)
 	switch msg := e.Message.(type) {
 	case *oracleproto.GossipedVotes:
-		// verify sig of incoming gossip vote, throw if verification fails
-		_, val := oracleR.ConsensusState.Validators.GetByAddress(msg.Validator)
-		if val == nil {
-			logrus.Debugf("validator: %v not found in validator set, skipping gossip", hex.EncodeToString(msg.Validator))
+		// get account and sign type of oracle votes
+		accountType := []byte{msg.Signature[0]}
```

Comment:
pull this out

Reply:
resolved in https://github.com/Switcheo/cometbft/pull/6/commits/0da46e4fc59d5032228febaf96b53a6917250976